src/styles/_catalog.scss

```
@@ -14,9 +16,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: color, transform $transition-duradtion;
```

```suggestion
    transition: color $transition-duradtion, transform $transition-duradtion;
```
Для color тоже укажи длительность